From 546e38962420b2597fcecf7607c3ec0a2d15bab4 Mon Sep 17 00:00:00 2001
From: mbrinkma <mbrinkma@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 7 Jan 2021 09:34:48 +0000
Subject: [PATCH 402/462] MANTIS 0041015 BuildManagement: make
 "clean"-buildstep optional for nightly build as it currently produces errors

git-svn-id: svn://localhost/SelfServiceBuild/trunk@733 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/release/makefile-jenkins | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SelfServiceBuild/Build/release/makefile-jenkins b/SelfServiceBuild/Build/release/makefile-jenkins
index 2215393..2eeec8d 100644
--- a/SelfServiceBuild/Build/release/makefile-jenkins
+++ b/SelfServiceBuild/Build/release/makefile-jenkins
@@ -153,9 +153,9 @@ build_application:  create build_cpp
 
 nb_application:     build_cpp sendMailZip
 
-nb_Platform:        clean init checkout set_props build_cpp sendMailZip
-nb_Platform_java:   clean init checkout set_props build_java sendMailZip
-nb_Platform_full_nomail:   clean init set_props syncBuildResources build_cpp build_maven build_java build_gradle build_installer kidlist idllist
+nb_Platform:        init checkout set_props build_cpp sendMailZip
+nb_Platform_java:   init checkout set_props build_java sendMailZip
+nb_Platform_full_nomail:   init set_props syncBuildResources build_cpp build_maven build_java build_gradle build_installer kidlist idllist
 nb_Platform_full:   nb_Platform_full_nomail sendMailZip
 
 build_mavenQBG:     $(SELF)
-- 
2.41.0.windows.1

